Tests/PostgresNIOTests/New/PSQLDataTests.swift

```diff
@@ -15,4 +15,10 @@ class PSQLDataTests: XCTestCase {
         XCTAssertNoThrow(emptyResult = try data.decode(as: String?.self, context: .forTests()))
         XCTAssertNil(emptyResult)
     }
+    
+    func testMetadataParsing() {
+        XCTAssertEqual(100, PostgresQueryMetadata(string: "SELECT 100")?.rows)
+        XCTAssertEqual(0, PostgresQueryMetadata(string: "SELECT")?.rows)
+        XCTAssertNil(PostgresQueryMetadata(string: "SELECT 100 100"))
+    }
```

Comment:
Would you mind creating a new test file for this test:
```
PostgresNIOTests/PostgresQueryMetadataTests.swift
```

Since it isn’t really related to PSQLData.

Reply:
Fixed!

Reply:
Awesome! Thanks!